internal/app/app_test.go

```diff
@@ -67,8 +67,7 @@ func TestWait(t *testing.T) {
 				args[i] = tc.applyArgs[i]
 			}
 
-			ctx := context.WithValue(context.Background(), proxy.DryRun, true)
-			a, err := app.New(ctx, app.WithAuthorizer(&app.MockAuthorizer{RejectAuth: tc.rejectAuth}))
+			a, err := app.New(context.Background(), app.WithAuthorizer(&app.MockAuthorizer{RejectAuth: tc.rejectAuth}), app.WithProxy(&app.MockProxy{ApplyError: tc.proxyApplyError}))
 			require.NoError(t, err, "New should have succeeded but didn't")
```

Comment:
I know it's not on the diff, but this is a setup action so we should have the require message prefixed with the `"Setup:"` tag